.azure-devops/graphitation-release.yml

```diff
@@ -1,6 +1,7 @@
 pr: none
 trigger:
   - main
+  - supermassive-v3
```

Comment:
So I believe this will need to be merged into main

Reply:
Good point, thanks.